modules/sap_hana/main.tf

```diff
@@ -80,12 +78,15 @@ resource "google_compute_instance" "gcp_sap_hana" {
     subnetwork         = "${var.subnetwork}"
     subnetwork_project = "${var.project_id}"
 
-    access_config {
```

Comment:
Has this change been tested for backwards compatibility? If the IP addresses of existing servers are going to be changed then that needs to be documented in the CHANGELOG.

Reply:
Ajay and Anil after discussion with Morgante have agreed upon to inform the client to upgrade the terraform version to 0.12 and above . Going forward they will have all modules with version 0.12 and above.